lib/main.dart

```diff
@@ -108,6 +118,23 @@ class _RandomWordsState extends State<RandomWords> {
   }
 }
 
+class Info extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text("Info"),
+      ),
+      body: const Center(
+          child: Text(
+        "Here you can add your words to the cart. \nYes!!! You can buy them. \n The future is now, wake up",
+        style: TextStyle(
+            fontSize: 14, color: Colors.black, fontWeight: FontWeight.normal),
```

Comment:
Doesn't text have a normal font weight by default?

Reply:
I'm sure it does (but @Nazarko12 correct me if I'm wrong)